backend/models/submission.js

```diff
@@ -0,0 +1,81 @@
+const mongoose = require("mongoose");
+
+const coordinates = new mongoose.Schema({
+  x: {
+    type: Number
+  },
+  y: {
+    type: Number
+  }
+});
+
+const option = new mongoose.Schema({
+  min: {
+    type: Number
+  },
+  max: {
+    type: Number
+  },
+  opt: {
+    type: String,
+    default: null
+  }
+});
+
+const answer = new mongoose.Schema({
+  type: {
+    type: String,
+    enum: [
+      "SHORT_ANSWER", //Yoptions.length = 0 | Xoptions.length = 1
+      "PARAGRAPHS", //Yoptions.length = 0 | Xoptions.length = 1
+      "MULTIPLE_CHOICE", //Yoptions.length = # of MC | Xoptions.length = 1
+      "CHECKBOXES", //Yoptions.length = # of boxes | Xoptions.length = 1
+      "FILE_UPLOAD", //Yoptions.length = 0 | Xoptions.length = 1
+      "CHECKBOX_GRID" //(Y|X)options.length = # of choices
+    ]
+  },
+  // SHORT_ANSWER
+  // PARAGRAPH
+  answerString: {
+    type: String
+  },
+  // MULTIPLE_CHOICE
+  // CHECKBOXES
+  answerArray: {
```

Comment:
@ahmedhamodi This should just be `[String]` instead of `[option]`?

We are storing an array of ids to represent the answers selected. If we need to access the "min"/"max" that a user can select we can compare to the form when generating the decision canvas.

Reply:
That is true - good catch. @annoyingpr can you make this change?

Reply:
Gotcha, just updated that and removed the options schema!